index2.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style2.css" />
+
+    <title>David Underwood</title>
+  </head>
+  <body>
+    <header>
+      <div class="header">
+        <img class="cell-info" src="Light.png" alt="cell phone info" />
+      </div>
+      <div class="bottom-section">
+        <div class="welcome">
+          <a href="https://github.com/david63011">Welcome</a>
+        </div>
+        <img class="union" src="Union.png" alt="union photo" />
+      </div>
+    </header>
+    <div class="main-section">
+      <section>
+        <img src="background.png" alt="green background" />
+        <div class="greeting">
+          <p class="hi">Hey there. I'm</p>
+          <p class="first-name">Mariane</p>
+          <p class="job">I am a software</p>
+          <p class="job-info">Developer</p>
+        </div>
+        <div class="information-box">
+          <h1 class="info">I can help you build a product,</h1>
+          <h1 class="info2">feature or website Look through</h1>
+          <h1 class="info3">some of my work and experience!</h1>
+          <h1 class="info4">If you like what you see and have</h1>
+          <h1 class="info5">a project you need coded</h1>
+          <h1 class="info6">don’t hestiate to contact me.</h1>
+        </div>
```

Comment:
- [ ] `required`
- [ ] its not best practice to break one word into chuncks according to to the [html&Css](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) sematic tags kindly put the pagragraph in a ptag